src/open_meteo_solar_forecast/open_meteo_solar_forecast.py

```diff
@@ -200,16 +200,36 @@ def gen_power(gti: float, t_amb: float, eff: float) -> int:
 
             Formulas:
             ---------
+                Assuming https://www.mdpi.com/2071-1050/14/3/1500 equation 1 + 2 as well as table 1
+                the formula:
+                
                 Tc=Ta + (G/Gnoct) * k
+                and
+                P=Pmax * (G/Gstc) * (1 + α * (Tc-Tstc)) * ηDC (p.509)
+                
+                must be:
+                
+                Tc=Ta + G * k
+                and
+                P=Pmax * G * (1 + α * (Tc-Tstc)) * ηDC (p.509)
```

Comment:
`P=Pmax * G * (1 + α * (Tc-Tstc)) * ηDC (p.509)`

Why change it to this? I am following up until here.